_includes/3d-secure-2.md

```diff
@@ -7,107 +7,112 @@
 
 ## 3-D Secure 2
 
-When dealing with card payments, 3-D Secure authentication of the
-cardholder is an essential topic. 3-D Secure 2 is an improved version of the
-old protocol, now allowing frictionless payments where transactions can be
-completed without input from the cardholder. Therefore, there are certain fields
-that should be included when implementing 3-D Secure 2. These are listed below
-and can be seen in the abbreviated request example below.
+When dealing with card payments, 3-D Secure authentication of the cardholder is
+an essential topic. 3-D Secure 2 is an improved version of the old protocol, now
+allowing frictionless payments where transactions can be completed without
+input from the cardholder. To increase the chances of this, there are certain
+fields that should be included in your request when implementing 3-D Secure 2.
 
-{:.code-view-header}
-**Request**
-
-```http
-POST /psp/{{ api_resource }}/payments HTTP/1.1
-Authorization: Bearer <AccessToken>
-Content-Type: application/json
-
-{
-    "{{ api_resource_field_name }}": {
-        "operation": "Purchase",
-        "intent": "Authorization",
-        "currency": "SEK",
-        "description": "Test Purchase",
-        "urls": {
-            "hostUrls": ["https://example.com"]
-        }, {% if api_resource == "creditcard" %}
-        "cardholder": {
-            "firstName": "Olivia",
-            "lastName": "Nyhuus",
-            "email": "olivia.nyhuus@payex.com",
-            "msisdn": "+4798765432",
-            "homePhoneNumber": "+4787654321",
-            "workPhoneNumber": "+4776543210",
-            "shippingAddress": {
-                "addressee": "Olivia Nyhuus",
-                "email": "olivia.nyhuus@payex.com",
-                "msisdn": "+4798765432",
-                "streetAddress": "Saltnestoppen 43",
-                "coAddress": "",
-                "city": "Saltnes",
-                "zipCode": "1642",
-                "countryCode": "NO",
-            },{% else %}
-        "payer": {
+```
```

Comment:
At the start of all fenced code examples, we need to declare which language is used. Why is `http` removed? The code example is not valid JSON anymore either.